components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/DefaultStepBasedSequenceHandlerTest.java

```diff
@@ -89,10 +88,9 @@
 import static org.testng.Assert.fail;
 import static org.wso2.carbon.identity.core.util.IdentityUtil.getLocalGroupsClaimURI;
 
-@PrepareForTest({
-                        FrameworkUtils.class, IdentityApplicationManagementUtil.class, ApplicationMgtSystemConfig.class,
-                        IdentityTenantUtil.class
-                })
+@PrepareForTest({FrameworkUtils.class, IdentityApplicationManagementUtil.class, ApplicationMgtSystemConfig.class,
+        IdentityTenantUtil.class, IdentityUtil.class})
+
```

Comment:
Remove redundant line

Reply:
Addressed with commit 12e680f42aedfa730f6f23a0c28f99c8a56868da in https://github.com/wso2/carbon-identity-framework/pull/3774 PR